makefile

```diff
@@ -6,18 +6,22 @@ SHELL := /bin/bash
 .PHONY: clean test integration consul etcd run-consul run-etcd example example-consul example-etcd ship dockerfile docker cover lint
 
 VERSION ?= dev-build-not-for-release
-LDFLAGS := '-X main.GitHash=$(shell git rev-parse --short HEAD) -X main.Version=${VERSION}'
+LDFLAGS := '-X containerbuddy.GitHash=$(shell git rev-parse --short HEAD) -X containerbuddy.Version=${VERSION}'
 
 ROOT := $(shell pwd)
+PACKAGE := github.com/joyent/containerbuddy
+GO15VENDOREXPERIMENT := 1
+export GO15VENDOREXPERIMENT
 
 COMPOSE_PREFIX_ETCD := exetcd
 COMPOSE_PREFIX_CONSUL := exconsul
 
+
 DOCKERMAKE := docker run --rm \
 	--link containerbuddy_consul:consul \
 	--link containerbuddy_etcd:etcd \
-	-v ${ROOT}/src/containerbuddy:/go/src/containerbuddy \
 	-v ${ROOT}/.godeps:/go/src \
```

Comment:
We're mounting volumes that don't exist like `.godeps` and we should be mounting the `vendor` directory so that we don't need to re-download.